Wrappers/Python/ccpi/viewer/CILViewer.py

```diff
@@ -37,6 +37,13 @@ def __init__(self, callback):
         #self.AddObserver('RightButtonPressEvent', self.OnRightMousePress, -0.5)
         #self.AddObserver('RightButtonReleaseEvent', self.OnRightMouseRelease, -0.5)
 
+        self._volume_render_pars = {
```

Comment:
The dialogs in the toolbar did not inherit the dark theme correctly. Probably this is due to not setting the parent of the dialog.